assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -43,6 +43,7 @@ const ANALYTICS_4_METRIC_TYPES = {
 	conversions: 'TYPE_INTEGER',
 	screenPageViews: 'TYPE_INTEGER',
 	engagedSessions: 'TYPE_INTEGER',
+	engagementRate: 'TYPE_INTEGER',
```

Comment:
See https://github.com/google/site-kit-wp/pull/6689#discussion_r1127029972

Reply:
Thanks, I've fix this on the #6689 branch and cherry-picked the commit here as well.